### PR TITLE
Csv uploader

### PR DESCRIPTION
**Pull request by sposerina**
#31 initial submission. Basic upload works with server side validation and saves csv to the filesystem. Still want to add client side verification and source (pet point, volgisitics, etc) detection.

### CODE REVIEW

**Comment by sposerina** [comment]
Finished the upload csv webpage functionality. Uploads are validated for file size, file extension, and proper csv column headers. Allows selecting multiple files for upload. Validation errors and successful uploads are displayed in the webpage. Uploads are saved under the /app/static/uploads folder in the container.

File: src/api_server/api_app.py [hunk 1]
```diff
@@ -1,21 +1,73 @@
...
+        flash('no file part', 'error')
+        return redirect(request.url)
+    
+    for file in request.files.getlist('file'):
+        if not allowed_file(file.filename):
+            flash('not a csv: ' + file.filename, 'error')
+            continue
+        try:
+            header = file.stream.readline().decode('utf-8').split(',')[1]
+            cleaned = header.strip()
```
Comment (urirot) [inline src/api_server/api_app.py]: I think the name "cleaned" can be more explanatory

File: src/api_server/api_app.py [hunk 2]
```diff
@@ -27,7 +79,7 @@ def listFiles():
 def executeScript(scriptName):
     print('Start executing script: ' + scriptName)
     try:
-        sample_script.run()
+        #sample_script.run()
```
[the remark this answers is not in the document]
Comment (sposerina) [reply src/api_server/api_app.py]: Was having trouble resolving module in local environment but will fix on next submission

File: src/api_server/api_app.py [hunk 3]
```diff
@@ -1,21 +1,73 @@
...
+        if not allowed_file(file.filename):
+            flash('not a csv: ' + file.filename, 'error')
+            continue
+        try:
+            header = file.stream.readline().decode('utf-8').split(',')[1]
+            cleaned = header.strip()
+            valid_src = True
+            if cleaned == 'Account.Name':
+                flash('Salesforce' + SUCCESS_MSG  + file.filename, 'info')
+            elif cleaned == 'Last.name..First.name':
```
[the remark this answers is not in the document]
Comment (sposerina) [reply src/api_server/api_app.py]: In the actual header

File: src/api_server/api_app.py (hunk 1 above)
Comment (sposerina) [reply src/api_server/api_app.py]: Agreed and will change to 'column_header'

**Pull request merged**